src/components/form/new.vue

```diff
@@ -318,7 +324,8 @@ export default {
       "Workflow warnings:",
       {
         "deletedFormExists": "There is a form with ID \"{value}\" in the Trash. If you upload this Form, you won’t be able to undelete the other one with the matching ID.",
-        "structureChanged": "The following fields have been deleted, renamed or are now in different groups or repeats. These fields will not be visible in the Submission table or included in exports by default."
+        "structureChanged": "The following fields have been deleted, renamed or are now in different groups or repeats. These fields will not be visible in the Submission table or included in exports by default.",
+        "oldEntityVersion": "Entities specification version “{value}” is not compatible with Offline Entities. Please use version 2024.1.0 or later."
```

Comment:
Could possibly say "We recommend using version 2024.1.0 or later..." to leave it more open ended?